knockout-tinymce.coffee

```diff
@@ -1,4 +1,5 @@
 (($, ko) ->
+  cache = ""
```

Comment:
Right, I was thinking that it wouldn't be a problem, but on closer inspection there is one case I could think of where it could be troublesome. A WeakMap would probably solve this, but requires a polyfill and hence a lot of additional code. A simpler solution would probably be to just have two variables:
cachedContent: string;
cachedInstance: Tinymce.Editor | Element;
the reason for this whole thing was that typing felt slow and the profiler showed that innerHTML was being called a lot, and a lot of those calls came from the getContent() != value check. But it should probably also be set in the update handler...